changelog_unreleased/javascript/13677-2.md

```diff
@@ -0,0 +1,20 @@
+#### Fix Idempotence for `break|continue|debugger` (#13677 by @liuxingbaoyu)
+
+<!-- Optional description if it makes sense. -->
+
+<!-- prettier-ignore -->
+```js
+// Input
+
+for (const f in {}) break// comment
+;
+
+// Prettier stable
+
+for (const f in {})
+  break; // comment
+
+// Prettier main
+
+for (const f in {}) break; // comment
+```
```

Comment:
````suggestion
<!-- prettier-ignore -->
```js
// Input
for (const f in {}) break// comment
;

// Prettier stable
for (const f in {})
  break; // comment

// Prettier stable (second format)
for (const f in {}) break; // comment

// Prettier main
for (const f in {}) break; // comment
```
````